argo-cd-apps/base/toolchain-member/toolchain-member-operator.yaml

```diff
@@ -9,6 +9,9 @@ spec:
           - nameNormalized
         generators:
           - clusters:
+              selector:
+                matchLabels:
+                  appstudio.redhat.com/toolchain-member: 'true'
```

Comment:
should you use the `k-components/assign-toolchain-member-role-to-local-cluster` patch instead?

Reply:
My understanding is that patch is meant for applying the label to the argocd local cluster secret for use in a single cluster dev env. I opted not to create a new k-component here, like backup, because it would only be applied to this appset.

Reply:
ok,  make sence, I reviewed too quickly